src/OrchardCore.Modules/OrchardCore.DataProtection.Azure/Startup.cs

```diff
@@ -29,8 +29,7 @@ public override void ConfigureServices(IServiceCollection services)
         if (!string.IsNullOrWhiteSpace(connectionString))
         {
             services
-                .AddSingleton(new BlobOptions())
-                .AddTransient<BlobOptionsSetup>()
+                .Configure<BlobOptions, BlobOptionsSetup>()
```

Comment:
I think PostConfigure is better name to represent the timing of the configuration

Reply:
See https://github.com/OrchardCMS/OrchardCore/pull/14759/files#r1405670950